radix-engine-stores/src/hash_tree/test.rs

```diff
@@ -366,8 +366,15 @@ fn change(
     substate_offset_seed: u8,
     value_hash_seed: Option<u8>,
 ) -> SubstateHashChange {
+    use crate::interface::DatabaseMapper;
+    use crate::jmt_support::JmtMapper;
+    let (node_id, module_id, substate_key) =
+        substate_id(node_id_seed, module_id, substate_offset_seed);
```

Comment:
This fixes a compile error in these tests... not sure how this wasn't caught on the previous PR?